lrpar/tests/build.rs

```diff
@@ -0,0 +1,57 @@
+use cfgrammar::yacc::{YaccKind, YaccOriginalActionKind};
+use glob::glob;
+use lrlex::LexerBuilder;
+use lrpar::CTParserBuilder;
+use std::{env, fs, io::Read, path::PathBuf};
+use yaml_rust::YamlLoader;
+
+fn main() -> Result<(), Box<std::error::Error>> {
+    let out_dir = env::var("OUT_DIR").unwrap();
+    for entry in glob("src/*.test")? {
+        match entry {
+            Ok(path) => {
+                if path.is_file() {
+                    // Parse test file
+                    let mut file = fs::File::open(&path)?;
+                    let mut s = String::new();
+                    file.read_to_string(&mut s)?;
+                    let docs = YamlLoader::load_from_str(&s).unwrap();
+                    let grm = &docs[0]["grammar"].as_str().unwrap();
+                    let lex = &docs[0]["lexer"].as_str().unwrap();
+                    let yacckind = match docs[0]["yacckind"].as_str().unwrap() {
```

Comment:
I wonder if we should make the user type out the full `YaccKind` e.g. `Original(YaccOriginalActionKind::NoAction)` and so on? Then it's one less thing to learn the rules about mapping the two.